sympy/series/tests/test_limits.py

```diff
@@ -282,7 +282,7 @@ def test_set_signs():
     assert limit(abs(sin(x + 1)), x, 0) == sin(1)
 
     # https://github.com/sympy/sympy/issues/9449
-    assert limit((Abs(x + y) - Abs(x - y))/(2*x), x, 0) == sign(y)
+    assert limit((Abs(x + y) - Abs(x - y))/(2*x), x, 0) == 1/sign(y)
 
```

Comment:
Could you also please add a test for the same. You could frame a test case from the failing cases pointed out in the issue description.